challenges/functions.js

```diff
@@ -7,26 +7,36 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume (c, d, cb) {
+  return cb(c, d);
+}
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+function add (a, b) {return a + b;}
```

Comment:
When using the implicit return you dont need to state return; however nice work on the one-liners